content/events/2024-01-10-mastering-bitcoin-whitepaper.md

```diff
@@ -0,0 +1,58 @@
+---
+date: '2024-01-01'
+title: 'Mastering Bitcoin: Whitepaper & Ground Rules'
+type: Doc
+---
+
+![Mastering Bitcoin](/mastering-bitcoin-whitepaper.jpg)
+
+Join our vibrant reading discussion group as we dive into <a href="https://www.amazon.com/Mastering-Bitcoin-Programming-Open-Blockchain/dp/1098150090?crid=3FFD1FN2H7TZF&keywords=mastering+bitcoin+3rd+edition&qid=1703963363&sprefix=mastering+bitcoin,aps,153&sr=8-1&ufe=app_do:amzn1.fos.006c50ae-5d4c-4777-9bc0-4513d670b6bc" target="_blank">Mastering Bitcoin</a>, the ultimate technical guide to understanding Bitcoin. Perfect for those eager to deeply grasp the intricacies of Bitcoin, this group combines a fun and supportive environment for learning complex technical concepts.
+
+Each week, we'll explore a different pair of chapters, enriching your self-reading experience. We'll provide thought-provoking questions ahead of each session to guide your reading and fuel engaging discussions. Our meetups are not just about discussing the book; they're about building a community of Bitcoin enthusiasts and builders.
+
+Discover, discuss, and delve into the world of Bitcoin with us. Your journey towards mastering Bitcoin starts here!
+
+## How It Works
+
+On your own time, read this week's chapter. Show up at this event (in-person or remotely) to discuss the chapter with other participants. Each session will follow this format:
+
+1. Introductions: participants are invited to introduce themselves (we will skip this if there is nobody new in the room).
+2. Queries: then, we will go around the room to quickly discern what questions people had, what people were confused about, etc. and take note of these on the whiteboard so we can discuss further.
```

Comment:
My nit: i like "queries" because it aligns with the single word headings but idc either way

Reply:
Yeah,  I was trying to be cool by making them all single-word. However, because the "post-meetup hangout" makes a lot of sense, that also violates the single word format. So I made the other one "gather questions".